recipe/gen_patch_json.py

```diff
@@ -1473,6 +1473,11 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
             _replace_pin("flake8 >=3.5", "flake8 >=4.0", record["depends"], record)
             _replace_pin("pytest >=3.5", "pytest >=7.0", record["depends"], record)
 
+        # pyzmq 23.0.0 broke zerorpc-python
+        # https://github.com/0rpc/zerorpc-python/issues/251
+        if record_name == "zerorpc-python" and record["version"] == "0.6.3":
```

Comment:
```suggestion
        if record_name == "zerorpc-python" and record["version"] == "0.6.3" and record["build_number"] == 0:
```

you must specify your build number, otherwise, future builds might get included.

Others have suggested other metrics, such as "timestamps", which are likely good, except for the fact that I don't know how to generate them.